llvm/test/CodeGen/RISCV/GlobalISel/instruction-select/bswap-rv64.mir

```diff
@@ -1,19 +1,22 @@
 # NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
 # RUN: llc -mtriple=riscv64 -mattr=+zbb -run-pass=instruction-select \
 # RUN:   -simplify-mir -verify-machineinstrs %s -o - \
-# RUN:   | FileCheck -check-prefix=RV64I %s
+# RUN:   | FileCheck %s
+# RUN: llc -mtriple=riscv64 -mattr=+zbkb -run-pass=instruction-select \
+# RUN:   -simplify-mir -verify-machineinstrs %s -o - \
+# RUN:   | FileCheck %s
```

Comment:
Does this need an extra line? It looks like it might fit on the last line

Reply:
Yep, done. Thanks